.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: Build/test
+on:
+  pull_request:
+    branches:
+      - "**"
+  push:
+    branches:
+      - "master"
+      - "main"
+jobs:
+  test_ubuntu:
```

Comment:
Based on the examples in your PR for other languages, I see that some languages test on just Ubuntu and some test on Ubuntu + macOS + Windows. Are there language regression cases that have only happened on platforms other than Ubuntu? I ask because macOS and Windows have a 10 and 2 minute usage multiplier, respectively.

Reply:
I am unaware of language regression cases that happen on platforms besides Ubuntu. I added MacOS testing since it was easy/convenient to do so and I saw the other tree-sitter repos testing all 3 platforms (even though it might not be completely necessary). Good note about the usage! After some [searching](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions), I believe "GitHub Actions usage is free for public repositories" - so it would only be an issue if we were to make this repo private.

Reply:
Yes, it is free but only up to a certain limit. With the action runtimes on this PR, we will use up about 6-7 minutes of the quota per PR. That is around 300 free PR actions a month on the Free tier, which I doubt we will exceed on this repo. However, if this repo moves to another organization or Antonio has other repos which use Actions, then it may become more relevant. This is because the free quota is per user, not per repo.

Fine to keep it, just wanted to explain the thinking.

Reply:
Wait jk. I am having so much trouble with this article wording. Public repos have unlimited for free. I've never struggled to understand three sentences more in my life.